src/Zicht/Bundle/UrlBundle/Aliasing/Doctrine/CreateAliasSubscriber.php

```diff
@@ -20,21 +20,38 @@ class CreateAliasSubscriber extends BaseSubscriber
     private $isHandling = false;
 
     /**
+     * @var array
+     */
+    private $records = [];
+
+    /**
      * @return array
      */
     public function getSubscribedEvents()
     {
         if ($this->enabled) {
-            return array(
+            return [
                 Events::postPersist,
                 Events::postUpdate,
                 Events::postFlush,
-            );
+            ];
         } else {
-            return array();
+            return [];
         }
     }
 
+    protected function addRecord($entity, array $action = [])
```

Comment:
consistency, vond het maar niks dat er protected en private door elkaar werden gebruikt en zag geen reden voor private (en geloof dat we afgesproken hadden dat voor bundels we protected zouden gebruiken zodat het extendable is)